docs/en/ecosystem/external-table/iceberg-of-doris.md

```diff
@@ -135,10 +167,14 @@ You can also create an Iceberg table by explicitly specifying the column definit
     - The order of the columns needs to be consistent with the Iceberg table
 - ENGINE needs to be specified as ICEBERG
 - PROPERTIES property.
-    - `iceberg.hive.metastore.uris`: Hive Metastore service address
     - `iceberg.database`: the name of the database to which Iceberg is mounted
     - `iceberg.table`: the name of the table to which Iceberg is mounted, not required when mounting Iceberg database.
-    - `iceberg.catalog.type`: the catalog method used in Iceberg, the default is `HIVE_CATALOG`, currently only this method is supported, more Iceberg catalog access methods will be supported in the future.
+    - `iceberg.catalog.type`: the catalog type used in Iceberg, the default is `HIVE` (simple name of `org.apache.iceberg.hive.HiveCatalog`). Iceberg built-in supports two catalogs with a simple name: `HIVE` and `HADOOP`. You should use the full class name for other implementations.
```

Comment:
How about add more built-in catalogs, such as JDBC, Glue catalog, for user convenience ?

User only need set full class name when using a custom catalog.